src/main/java/de/thomas_oster/visicut/misc/Helper.java

```diff
@@ -300,6 +305,10 @@ else if (isMacOS()) {
         String line = r.readLine();
         while (line != null)
         {
+          if (appImagePath != null && line.equals("VISICUTBIN = \"\"")) {
+            System.err.println("APPIMAGE=" + appImagePath);
```

Comment:
debug output should use `Logger`.

Reply:
This is an oversight, I'll get rid of it.